### DISCUSSION TITLE
Extending hindsight to 1 minute

### DISCUSSION THREAD

**Discussion by MtnXfreeride** [gopro/labs#611] [Feature Requests]
Would labs firmware be able to extend hindsight from the 30 second maximum to 1 minute (or longer)?   It would make the feature more useful for fishing or nearly any activity to not have to rush to press record.

**Comment by dnewman-gpsw** [comment]
This is unlikely, as Hindsight is very memory intensive, I doubt the camera could buffer than much data.

**Reply by MtnXfreeride** [reply]
Thanks, I wondered if it was really using memory or just writing to the card.. hard to believe even 30 seconds can fit in memory as a buffer.. that's a lot of memory.. so I figured I would ask.

**Comment by CarloCesenatico** [comment]
Buongiorno Io avrei invece un una richiesta contraria cioè se è possibile ridurre il senno di poi a tre o cinque secondi in quanto metto la GoPro puntata su abbeveratoio di animali con l'avvio della registrazione col movimento e anche 15 secondi sono troppi. Grazie

**Reply by dnewman-gpsw** [reply]
hS5 - Hindsight 5 seconds
hS10 - Hindsight 10 seconds

These are more experimental, may not work.

**Reply by CarloCesenatico** [reply]
Scusa ma ho scaricato solo ieri GoPro Lab e l'app QR Control e sapendone poco ti chiedo dove è da mettere hS5 nella app per diminuire il  tempo del senno di poi?
Ti faccio altre due domande: è possibile avere sempre spento l'LCD? Nella app si può selezionare: sempre acceso oppure 1/2/3 minuti ma non spento.
Seconda domanda: posso alimentare la GoPro 11 con una batteria da 7,2 volt 6 Ah?
Grazie scusa il disturbo
Carlo

**Reply by dnewman-gpsw** [reply]
![image](https://github.com/gopro/labs/assets/10856084/d229d487-af24-4eff-9d43-463cf7f68e98)
![image](https://github.com/gopro/labs/assets/10856084/61df6c53-df9a-4204-86e3-a411c0f5a5e7)

No, it is not possible to have LCD off all the time.

Powering GoPro 11 with a 7.2 volt 6 Ah battery: You would need a voltage regulator to convert it to 5V over USB, but that should will work.

**Comment by CarloCesenatico** [comment]
Un'altra domanda: una volta settata la GoPro che registra appena vede un movimento, la registrazione si interrompe quasi subito in assenza di movimento. E' possibile posticipare di 15/30 secondi lo stop?  Così se un animale rimane fermo per qualche secondo non si interrompe la registrazione. Grazie. Carlo

**Reply by dnewman-gpsw** [reply]
Yes, That is what thehold time is for.  
![image](https://github.com/gopro/labs/assets/10856084/b11d448e-e352-4849-86b1-cecd280e65c0)

**Comment by CarloCesenatico** [comment]
Ciao David Ieri la mia GoPro Hero 11 utilizzata con il firmware Labs ha
smesso di leggere i video registrati, non effettua più il play. Può essere
colpa del firmware Labs? Ho fatto il reset della GoPro ma continua ad avere
il problema. Probabilmente dovrò sentire con l'assistenza.
Grazie
Carlo Godio

Il giorno sab 11 nov 2023 alle ore 18:40 David Newman <
***@***.***> ha scritto:

**Reply by dnewman-gpsw** [reply]
I don;t understand this error. Please record a video of the camera's error.

**Comment by CarloCesenatico** [comment]
Semplicemente dopo aver registrato dei video quando li voglio rivedere la
gopro mi dice che la scheda di memoria e vuota mentre se guardo la scheda
al computer le registrazioni ci sono e le vedo normalmente.

Il Gio 16 Nov 2023, 18:46 David Newman ***@***.***> ha
scritto:

**Reply by dnewman-gpsw** [reply]
Have you used? https://gopro.github.io/labs/control/basename/

If you are renaming or changing chapter size (or some older camera models) the files can't playback on camera.  This is in the documentation.  It is not a Labs error.

**Comment by CarloCesenatico** [comment]
No volutamente non ho cercato di rinominare i file. Probabilmente l'ho
fatto involontariamente perché in effetti i file sono nominati:
yymmdd-GX...
Ho reimpostato la gopro ma i file rimangono nominati come sopra. È stato
qualcosa che ho impostato involontariamente con il Qr code?

Il Gio 16 Nov 2023, 20:11 David Newman ***@***.***> ha
scritto:

**Reply by dnewman-gpsw** [reply]
Yes, user error. Use !RESET to reset Labs.  New captures will not have changed names.

**Comment by CarloCesenatico** [comment]
Ho fatto un reset ma ancora i file non sono nominati correttamente

Il Gio 16 Nov 2023, 22:08 David Newman ***@***.***> ha
scritto:

**Reply by dnewman-gpsw** [reply]
Old files will not change, only new files/captures will.

**Comment by CarloCesenatico** [comment]
Certo ho anche formattato la scheda per non avere file vecchi. Però ancora
non legge i nuovi file la gopro

Il Gio 16 Nov 2023, 23:20 David Newman ***@***.***> ha
scritto:

**Reply by dnewman-gpsw** [reply]
Then you have not successful reset Labs.

Scan 
![image](https://github.com/gopro/labs/assets/10856084/fed30e02-a86c-4e7f-a1f7-3ff3765467d5)

Please shutter three time rapid immediately after.  You must see "approved" otherwise you haven't performed a reset.

**Comment by CarloCesenatico** [comment]
Con il tuo qr code si e' resettata E adesso funziona bene. Con il mio la
gopro aveva accettato il reset, ma i file non erano visibili.
Mi piacerebbe sapere cosa ho fatto per combinare tutto questo pasticcio.
Ti ringrazio sei stato molto paziente e professionale se vieni in Italia ti
devo una cena. Di nuovo grazie ciao Carlo Godio

Il Gio 16 Nov 2023, 23:34 David Newman ***@***.***> ha
scritto:

**Reply by dnewman-gpsw** [reply]
If you use https://gopro.github.io/labs/control/metadata/ on any of those older files, it will show which Labs features you had activated.

Here is my camera, in green are the Labs extensions I have enabled.
![image](https://github.com/gopro/labs/assets/10856084/f1c85388-52cc-422a-b57f-5bd17cb36bf2)
